searchlib/src/vespa/searchlib/query/tree/stackdumpquerycreator.h

```diff
@@ -45,8 +47,26 @@ class StackDumpQueryCreator {
         return builder.build();
     }
 
-private: 
-    static Term * createQueryTerm(search::SimpleQueryStackDumpIterator &queryStack, QueryBuilder<NodeTypes> & builder, vespalib::stringref & pureTermView) {
+private:
+    static void populateMultiTerm(search::SimpleQueryStackDumpIterator &queryStack, QueryBuilderBase & builder, MultiTerm & mt) {
+        for (uint32_t i(0); i < mt.getNumTerms(); i++) {
+            queryStack.next();
```

Comment:
need to check for errors and end-of-stack (also, we might want to roll back to a deterministic error state in the next() function in case of en exception, just in case someone does something like this in the future...)

Reply:
Fixed